resources/containers/unsaved-changes-warning.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Modal from "react-modal";
+import modal_styles from "./styles/delete-confirmation.module.css";
```

Comment:
Duplicated CSS

If you want to have a generic modal styles, create a `BaseModal` resource, and le tthe other use it.

The point is save Styling.
Here we add the style again and we save on logic. 


```
<Modal
      isOpen={isOpen}
      onRequestClose={onRequestClose}
      className={modal_styles.modal}
      overlayClassName={modal_styles.overlay}
    >
     { children }
     </Modal>
     ```